### ISSUE TITLE
Bypass master pasword requirement after adding fingerprint

### ISSUE THREAD

**Issue by fredgolightly** [bitwarden/android#2776]
### Steps To Reproduce

I added a new fingerprint to my Pixel to unlock it with. Opening Bitwarden after I was told biometrics were disabled pending verification of master password, and could log in with pin.

Opened settings and disabled then re-enabled biometrics in Bitwarden, verified my fingerprint, and biometric unlocking was enabled, no master password required.

Either the master pw needs to be required as stated or the warning stating this needs clarifying

### Expected Result

Expect to have to enter master password as told

### Actual Result

Didn't need master password

### Operating System

Android

### Operating System Version

14 beta

### Device

Pixel 5

### Build Version

2023.8.0

### Beta

- [ ] Using a pre-release version of the application.

**Comment by Krychaz** [comment]
Hi there,

Thank you for your report! 

I was able to reproduce this issue, and I have flagged this to our engineering team.

If you wish to add any further information/screenshots/recordings etc., please feel free to do so at any time - our engineering team will be happy to review these.

Thanks once again!

**Comment by vvolkgang** [comment]
Issue migrated to https://github.com/bitwarden/mobile/issues/2776

**Comment by fredgolightly** [comment]
Hi,
I've just added a new face for facial recognition on my Pixel 9 Pro running Android 16 and this issue still persists - I was told, upon opening the BW app, that biometric unlocking was disabled on the device pending reentry of the master password. After logging in with my pin I was then able to re-enable biometric unlock without ever unlocking with the master password.

This is on 2025.9.0 (20692)

**Comment by SaintPatrck** [comment]
Hi @fredgolightly,

The behavior you're describing is completely normal and is how Android's security is designed to work to protect your vault.

- **Why it happens:** For your security, Android automatically disables biometric unlock anytime the saved fingerprints or faces on your device are changed. This is a crucial feature to prevent someone who gets ahold of your phone from adding their own fingerprint and accessing your apps.
 
- **Why you had to log in:** Because the biometric key was disabled by Android, Bitwarden required you to re-authenticate with your master password to prove your identity and unlock your vault.
 
- **Re-enabling Biometrics:** Once you successfully logged in with your master password, the app had securely verified your identity. It could then proceed to set up biometric unlock again for the new set of faces/fingerprints on your device.
 
In short, the system worked exactly as it should. It detected a change in your device's security settings and correctly prompted you for your master password to keep your vault secure.
